src/components/common/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import { deleteToken, getToken } from 'utils/localStorage';
+
+import BalckButton from './BalckButton';
```

Comment:
혼자 상대경로라 아쉬워요 
함께 절대경로로 맞춰주시면 좋을 것 같습니다
```suggestion
import BalckButton from 'components/common/BalckButton';
```